utilities/sanitizeTrend.js

```diff
@@ -1,5 +1,13 @@
-const sanitizeTrend = (rawTimeline) => {
-  return JSON.parse(rawTimeline).default.timelineData.map((point) => {
+module.exports = (rawTimeline) => {
+  let parsedTimeline = {};
+
+  try {
+    parsedTimeline = JSON.parse(rawTimeline);
+  } catch (error) {
+    throw error;
```

Comment:
I believe throwing an error will shut down the code.  So as written the server will halt if the try block receives rawTimeline as anything besides a string containing JSON.  If we alter the catch block to set parsedTimeline to rawTimeline, then we can handle the other common case where rawTimeline has somehow shown up pre-parsed.  Obviously, there are more cases that will still muck things up, but this lets us cover 2 common cases rather than 1.